HWTS-3/task-1.ts

```diff
@@ -0,0 +1,23 @@
+interface ivehicle {
+    getDetails(): string;
+    start(): string;
+}//1
+abstract class vehicle implements ivehicle {
```

Comment:
```suggestion
abstract class Vehicle implements IVehicle {
```
Классы тже называются с большой буквы